balena.env

```diff
@@ -0,0 +1,4 @@
+
+export BASE_URL=balena-cloud.io
```

Comment:
Still no, sorry :smiley:. It's `.com`, not `.io`: https://api.balena-cloud.com

This also suggests you're not testing these, since this won't work with the wrong name. You definitely need to test these scripts before we merge this.

Reply:
I've tested all the commands in the README.md. Apologies for that, I initially only intended to open an issue at first, then just decided it'd be easier to bang out the change myself!